docs/api/api_blueprint.apib

```diff
@@ -1464,136 +1396,157 @@ Create a new event using a `name`, `starts-at`, `ends-at`, `timzone` and an opti
+                    "identifier": "b8324ae2"
+                },
+                "type": "event",
+                "id": "1",
                 "links": {
-                  "self": "/v1/events/12/relationships/discount-codes",
-                  "related": "/v1/events/12/discount-codes"
+                    "self": "/v1/events/1"
                 }
-              }
             },
-            "attributes": {
-              "payment-country": null,
-              "paypal-email": null,
-              "code-of-conduct": "",
-              "can-pay-by-bank": false,
-              "schedule-published-on": null,
-              "payment-currency": null,
-              "organizer-description": "",
-              "xcal-url": null,
-              "original-image-url": null,
-              "topic": null,
-              "onsite-details": null,
-              "organizer-name": null,
-              "large-image-url": null,
-              "timezone": "UTC",
-              "deleted-at": null,
-              "ticket-url": null,
-              "can-pay-by-stripe": false,
-              "location-name": null,
-              "privacy": "public",
-              "state": "draft",
-              "latitude": null,
-              "is-tax-enabled": false,
-              "can-pay-onsite": false,
-              "type": null,
-              "starts-at": "2017-05-31T18:30:00+00:00",
-              "searchable-location-name": null,
-              "description": "",
-              "pentabarf-url": null,
-              "logo-url": null,
-              "is-ticketing-enabled": false,
-              "event-url": null,
-              "ical-url": null,
-              "can-pay-by-paypal": false,
-              "is-map-shown": true,
-              "is-sponsors-enabled": false,
-              "name": "New Event",
-              "sub-topic": null,
-              "icon-image-url": null,
-              "thumbnail-image-url": null,
-              "created-at": "2017-06-11T06:23:39.513383+00:00",
-              "longitude": null,
-              "can-pay-by-cheque": false,
-              "bank-details": null,
-              "cheque-details": null,
-              "is-sessions-speakers-enabled": false,
-              "identifier": "b8324ae2",
-              "ends-at": "2017-06-04T18:30:00+00:00"
+            "jsonapi": {
+                "version": "1.0"
             },
-            "type": "event",
-            "id": "12",
             "links": {
-              "self": "/v1/events/12"
+                "self": "/v1/events/1"
             }
-          },
-          "jsonapi": {
-            "version": "1.0"
-          },
-          "links": {
-            "self": "/v1/events/12"
-          }
         }
 
 
 ## Event Details [/v1/events/{event_identifier}]
 + Parameters
-    + event_identifier: b8324ae2 (string) - identifier of the event. (Can also be the event ID)
+    + event_identifier: 1 (string) - identifier or event id of the event. (b8324ae2 is an example of identifier)
```

Comment:
As identifiers are random, and dredd uses this example value for testing the API, the example values have to be event id. There's no easy way around this :)